stackslib/src/net/api/getattachmentsinv.rs

```diff
@@ -96,11 +96,13 @@ impl HttpRequest for RPCGetAttachmentsInvRequestHandler {
             if key == "index_block_hash" {
                 index_block_hash = StacksBlockId::from_hex(&value).ok();
             } else if key == "pages_indexes" {
-                if let Ok(pages_indexes_value) = value.parse::<String>() {
-                    for entry in pages_indexes_value.split(',') {
-                        if let Ok(page_index) = entry.parse::<u32>() {
-                            page_indexes.insert(page_index);
-                        }
+                #[allow(clippy::expect_used)]
+                let pages_indexes_value = value
+                    .parse::<String>()
+                    .expect("parse from Cow<str> is always safe");
```

Comment:
What about `let pages_indexes_value = value.to_string()` and we can avoid the `expect()`?

Reply:
Ah, yes, that's definitely better. Thanks!